pkg/devicemapper/devmapper_log.go

```diff
@@ -5,17 +5,31 @@ package devicemapper
 import "C"
 
 import (
+	"fmt"
 	"strings"
+
+	"github.com/Sirupsen/logrus"
 )
 
+// dmLogLevel gates what level of libdm's debugging output will be forwarded
+// through logrus. libdm uses logging *very* liberally and most people using
+// --debug [or Docker] aren't going to be debugging libdm bugs. You can enable
+// this by setting --storage-opt="dm.log_level=X".
+var dmLogLevel = LogLevelErr
```

Comment:
I ran this a few times and it didn't seem to spam the logs too much (and I don't like ignoring errors because it causes you to ignore some `_do_dm_ioctl` errors) but yeah, I'll change it back to the old default.